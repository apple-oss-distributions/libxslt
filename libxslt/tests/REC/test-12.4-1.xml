<doc xmlns="s:def">
    <elem attr="value" xmlns:äöü="uri"/>
    <text>text</text>
    <!-- comment -->
    <?pi content?>
</doc>
