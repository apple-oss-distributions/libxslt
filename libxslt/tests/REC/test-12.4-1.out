<?xml version="1.0"?>
<result>
  <document>id1</document>
  <element>id2</element>
  <attribute>id3</attribute>
  <namespace>id2ns</namespace>
  <namespace>id2nsC3A4C3B6C3BC</namespace>
  <text>id4</text>
  <comment>id5</comment>
  <processing-instruction>id6</processing-instruction>
</result>
